woocommerce/woocommerce-gutenberg-products-block
# Store Customization > Ensure the Just Arrived Full Hero pattern can have an AI selected image assigned to it

## What

Update the Just Arrived Full Hero pattern to ensure that:

* AI-managed images can be assigned to it.
* Add a background dim to the cover image to prevent readability issues depending on what image the AI assigned to the pattern.

## Why



These changes are required to ensure this pattern can work as expected within the Pattern Assembler and is correctly displayed independent of the images assigned to it by AI.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Access the JN install with the credentials provided here > p1696409554974299-slack-C053716F2H2
2. Create a new post
3. Insert the **Just Arrived Full Hero** pattern
4. Make sure it is correctly rendered both on the editor side and on the front end.

PS: the background image doesn't necessarily have to match what is displayed in the screenshot below; those are dynamically changed depending on the business description.


## Screenshots or screencast



| Before | After |
| ------ | ----- |
| <img width="1207" alt="Screenshot 2023-10-06 at 09 55 46" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/a79aa838-1df2-470a-a00b-00776146e87a"> | <img width="1208" alt="Screenshot 2023-10-06 at 09 55 12" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/94170c78-463b-4eb9-be32-33dea06f1261"> |

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Ensure the Just Arrived Full Hero pattern can have an AI-selected images assigned to it and add a background dim
